src/Versioning.js

```diff
@@ -16,6 +16,10 @@ export default function DetermineTargetsFromConfig(config?: BrowserListConfig):
     return browserslist(config);
   }
 
+  if (config && typeof config === 'string') {
+    return browserslist(config.split(',').map(item => item.trim()));
```

Comment:
Doesn't `browserslist` already support a string input? Is there a need to split the string?

Reply:
Yes you are right, according to https://www.npmjs.com/package/browserslist#js-api it can be string or array